src/main/resources/rules/HTL-5.md

```diff
@@ -0,0 +1,25 @@
+Always Place HTL Attributes After the Ones that are Part of the Markup
```

Comment:
This description is actually for different rule!

Reply:
changed to: Always try to re-use existing conditions, so the code is more readable.